ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/NtiUnderConsideration/Close.cshtml.cs

```diff
@@ -70,19 +73,28 @@ public async Task<IActionResult> OnPostAsync()
 		{
 			try
 			{
+				var status = Request.Form["status"];
+				var success = int.TryParse(status, out var result);
+				if (!success)
+				{
+					ModelState.AddModelError("options", "Please select a reason for closing NTI under consideration");	
```

Comment:
You can use the required flag on the radio item to make a radio list mandatory

Reply:
I'm leaving it for now